README.md

```diff
@@ -1 +1,8 @@
-# microverse_project_review_lesson
\ No newline at end of file
+# microverse_project_review_lesson
+
+
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Good job! adding the readme file to your project 👍 

Please, kindly make the following changes to your readme file to follow best practices🤗
- Adding a proper description of your project
- Links to your social media handles
- Give a list of languages the project was built with
- Contributions and acknowledgments.

This will make your readme file look more professional and explanatory to the anyone that accesses the project👏.